examples/example-vite-react-sql/src/typescript/models.gen.ts

```
@@ -0,0 +1,170 @@
+    direction: Direction;
+}
+
+// Type definition for `dojo_starter::models::Direction` enum
+export enum Direction {
+    Left,
+    Right,
+    Up,
+    Down,
+}
+export namespace Direction {
+    export function toString(d: Direction): string {
```

Biome (1.9.4)

[error] Do not shadow the global "toString" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)